src/components/BlogPost/styles.css

```diff
@@ -1,3 +1,26 @@
 .blog-post-wrapper {
-  background-color: gray;
+  border: 1px solid lightgray;
+  width: 31.25rem;
+  margin-top: 1.25rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-shadow: 1px 1px 2px lightgrey;
 }
+
+.blog-post-wrapper h1 {
+  text-transform: uppercase;
+  letter-spacing: 1.5px;
+  font-size: 1.35rem;
+  margin-top: 1rem;
+  margin-bottom: 0;
+  font-weight: 700;
+}
+
+.blog-post-wrapper h2 {
+  font-size: .85rem;
+  color: #b4cbd6;
+  margin-top: .3rem;
+  font-weight: 600;
+  letter-spacing: 1px;
+}
```

Comment:
Same comment here, add a blank line to this file.  I won't harp on this too hard in the future but worth noticing that github leaves an indicator that this is weird that you don't have one. I'll link an article  about it but its Javascript specific.  Basically, the way JS gets compiled means that instructions can combine if theres not a blank space between them.  9999/10000 times modern browsers would not make this mistake. Still worth noting

https://evanhahn.com/newline-necessary-at-the-end-of-javascript-files/#:~:text=It%20comes%20from%20the%20C,up%20on%20the%20same%20line.